src/pages/public/Assistenter/components/CityCard/Citycard.tsx

```diff
@@ -1,34 +1,56 @@
-import React, { useState } from "react";
+import React from "react";
 import ApplyReg from "../ApplyReg";
-import { City, CityButton } from "./CityButton";
 
-const Citycard = () => {
-  const [city, setCity] = useState<City>(City.TRONDHEIM);
+interface TabProps {
+  divisions: DivisionList[];
+}
 
-  return (
-    <div className="my-8 flex-col w-screen flex items-center justify-center w-1/2 mx-auto">
-      <span className="border-solid border-2 border-vektor-darblue align-items-center">
-        <div className="divide-y divide-vektor-darblue bg-white">
-          <div className="text-vektor-darblue">
-
-            <span className="flex justify-center">
-              {Object.values(City).map((cityValue) => (
-                <CityButton
-                  onChooseCity={setCity}
-                  city={cityValue}
-                  isActive={city === cityValue}
-                />
-              ))}
-            </span>
-          </div>
-          <div className="text-center">
+interface DivisionList {
+  name: string;
+  number: number;
+}
 
-            {/* Use ApplyCard when users can apply and NoApllyCard when it is locked */}
-            <ApplyReg cities={city} />
-          </div>
+const Tabs = ({
+  divisions,
+}:TabProps): JSX.Element => {
+  const [openTab, setOpenTab] = React.useState(1);
+  return (
+    <>
+      <div className="w-full">
+        <ul className="flex flex-wrap text-sm font-medium text-center text-gray-500 border-b border-gray-200 dark:border-gray-700 dark:text-gray-400" role="tablist">
+          {divisions.map((data) => {
+            return (
+              <li className="-mb-px mr-0 last:mr-0 flex-auto text-center">
+                <a
+                  className={`text-base font-bold px-1 py-3 block line leading-normal hover:border-t-2 hover:border-r-2 hover:border-l-2 hover:text-vektor-blue ${openTab === data.number ? `text-vektor-darblue bg-white border-t-2 border-r-2 border-l-2` : `text-vektor-darblue `}`}
```

Comment:
Sonarcloud suggests using `nested template literals`.
Example:
![image](https://user-images.githubusercontent.com/46197518/197869978-0db238d6-968d-4e1e-a91b-5c5391f4f15f.png)

I think it would make the code a lot easier to read!